Link to the tutorials in English by Krstev, Laporte, Maurel

eric-laporte opened the pull request:
Link to the tutorials in English by Krstev, Laporte, Maurel (issue #29)

eric-laporte committed:
Link to the tutorials in English by Krstev, Laporte, Maurel

Link to the tutorials in English by Krstev, Laporte, Maurel (issue #29)

martinec commented on pages/02.blog/03.announcements/tutorials-in-English/post.fr.md:
```diff
@@ -0,0 +1,32 @@
+
+- [Préparation](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_install.pdf)
+- [Diaporama](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER.pdf)
+- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```
Even if possible, the code repository is not designed to version/host binary documents, including .PDFs, there are sites such as https://www.docdroid.net (you can take another one) that allow to share PDFs efficiently , see e.g: https://docdro.id/3QHeKxj

In this way: 

- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```
- [Exercices](https://unitexgramlab.org/blog/announcements/tutorials-in-english/Unitex_NER_exercises.pdf)
```

May be:

- [Exercices](https://docdro.id/3QHeKxj)
```
- [Exercices](https://docdro.id/3QHeKxj)
```

martinec committed:
Merge branch 'master' into master

eric-laporte committed:
store pdf files in another server

Updating links because pdf files will be stored in a server out of unitexgramlab.org.

eric-laporte committed:
Merge pull request #1 from eric-laporte/add-tutorials

store pdf files in another server
